src/io/aleph/dirigiste/Executor.java

```diff
@@ -389,13 +384,13 @@ private Stats updateStats() {
         return new Stats
             (_metrics,
              _numWorkers.get(),
-             _utilizations.getAndSet(new Stats.UniformDoubleReservoir()).toArray(),
-             _taskArrivalRates.getAndSet(new Stats.UniformDoubleReservoir()).toArray(),
-             _taskCompletionRates.getAndSet(new Stats.UniformDoubleReservoir()).toArray(),
-             _taskRejectionRates.getAndSet(new Stats.UniformDoubleReservoir()).toArray(),
-             _queueLengths.getAndSet(new Stats.UniformLongReservoir()).toArray(),
-             _queueLatencies.getAndSet(new Stats.UniformLongReservoir()).toArray(),
-             _taskLatencies.getAndSet(new Stats.UniformLongReservoir()).toArray());
+             _utilizations.toSortedArray(),
+             _taskArrivalRates.toSortedArray(),
+             _taskCompletionRates.toSortedArray(),
+             _taskRejectionRates.toSortedArray(),
+             _queueLengths.toSortedArray(),
+             _queueLatencies.toSortedArray(),
+             _taskLatencies.toSortedArray());
```

Comment:
I think the ring buffer is a better choice, but I keep worrying that something in Dirigiste _depends_ on resetting the stats every control period, which we no longer do after this change. Gahh.

Reply:
> I think the ring buffer is a better choice, but I keep worrying that something in Dirigiste depends on resetting the stats every control period, which we no longer do after this change. Gahh.

I can a see a reason for the _queueLatencies and _taskLatencies__ where their values
depends on the number of `acquire/release` performed.
But not for the others ones.